Resources/Prototypes/Entities/Objects/Misc/handy_flags.yml

```
@@ -0,0 +1,167 @@
+- type: entity
+  parent: BaseItem
+  id: BlankHandyFlag
+  name: blank handy flag
+  description: "Some piece of white cloth wound on a stick. Definitely not handy flag of France."
+  components:
+  - type: Sprite
+    sprite: Objects/Misc/Handy_Flags/blank_handy_flag.rsi
+    state: icon
+  - type: Item
+    sprite: Objects/Misc/Handy_Flags/blank_handy_flag.rsi
+
+- type: entity
+  parent: BaseItem
+  id: NTHandyFlag
+  name: Nanotrasen handy flag
+  description: "Glory to NT! Wait, they really made a handy flag for a corporation?"
+  components:
+  - type: Sprite
+    sprite: Objects/Misc/Handy_Flags/NT_handy_flag.rsi
+    state: icon
+  - type: Item
+    sprite: Objects/Misc/Handy_Flags/NT_handy_flag.rsi
+
+- type: entity
+  parent: BaseItem
+  id: SyndieHandyFlag
+  name: Syndicate handy flag
+  description: "For truly rebel patriots. Death to NT!"
+  components:
+  - type: Sprite
+    sprite: Objects/Misc/Handy_Flags/syndie_handy_flag.rsi
+    state: icon
+  - type: Item
+    sprite: Objects/Misc/Handy_Flags/syndie_handy_flag.rsi
+    
+- type: entity
+  parent: BaseItem
+  id: LGBTQHandyFlag
+  name: LGBTQ handy flag
+  description: "Be gay do crime handy flag"
+  components:
+  - type: Sprite
+    sprite: Objects/Misc/Handy_Flags/lgbtq_handy_flag.rsi
+    state: icon
+  - type: Item
+    sprite: Objects/Misc/Handy_Flags/lgbtq_handy_flag.rsi
+    
+- type: entity
+  parent: BaseItem
+  id: PirateHandyFlag
+  name: Pirate handy flag
+  description: "Holding it in your hands, show these carp that you're not kidding."
```

```suggestion
  id: PirateHandyFlag
  name: pirate handy flag
  description: Hold it in your hands to show these carp that you're not kidding.
```
descs dont need quotation marks around them + dont capitalize pirate imo
descriptions for most of these read pretty weirdly imo but ill just point them out as i go